docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,188 @@ Interventions
+	vitamin A deficiency prevalence. This is a limitation caused by the low 
+	quality evidence regarding the relative risk of vitamin A fortification on 
+	vitamin A prevalence. However, on average, increasing coverage of vitamin 
+	A fortification will decrease VAD.
+
+To model the uncertainty in this estimate, the above RR should be drawn from a
+`lognormal <https://en.wikipedia.org/wiki/Log-normal_distribution>`_
+distribution with median = 2.22, 2.5\ :superscript:`th`-percentile = 0.95, and
+97.5\ :superscript:`th`-percentile = 5.26. This distbibution can be created
+using `SciPy's lognorm function
+<https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html>`_
+as follows:
+
+.. code-block:: Python
+
+  from numpy import log
+  from scipy.stats import norm, lognorm
+
+  # median and 0.975-quantile of lognormal distribution for RR
+  median = 2.22
+  q_975 = 5.26
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  mu = log(median) # mean of normal distribution for log(RR)
+  sigma = (log(q_975) - mu) / q_975_stdnorm # std dev of normal distribution for log(RR)
+
+  # Frozen lognormal distribution for RR, representing uncertainty in our effect size
+  # (s is the shape parameter)
+  rr_distribution = lognorm(s=sigma, scale=median)
+
+.. note::
+
+	I copied this from Nathaniel's documentation for the folic acid RR, but I 
+	think that the same approach is appropriate. Perhaps we can eventually 
+	create a separate page that lists similar strategies that we can reference 
+	via links.
+
+Additionally, as described in the research considerations above, the 
+intervention effect is dependent on age and time since intervention coverage.
+
+For simulants covered by *baseline coverage*, the effect of the intervention 
+is applied continuously from the start of the simulation to the end of the 
+simulation and only depends on age, such that:
+
+.. code-block:: Python
+
+  if age_i < 0.5:
+  	rr_i = 1 + (rr - 1) * a_i / 0.5
+  else:
+  	rr_i = rr
+
+For simulants covered by the *intervention scale-up*, the effect of the 
+intervention will scale up linearly from zero at the start of coverage to the 
+full effect after six months of coverage and does *not* depend on age (
+research assumption), such that:
+
+.. code-block:: Python
+
+  if coverage_time_i < 0.5:
+  	rr_i = 1 + (rr - 1) * coverage_time_i / 0.5
+  else:
+  	rr_i = rr
+
+.. note::
+
+	Any ideas from the research team on how we might want to combine
+	the impact of age_i < 0.5 and coverage_time_i < 0.5 here?
+
+Where,
+
+	- **age_i** = age of simulant in years
+
+	- **coverage_time_i** = time since start of intervention coverage for a simulant in years
+
+	- **rr_i** = relative risk to be applied to an individual simulant
+
+	- **rr** = relative risk for intervention, sampled as described above based on a random number between 0 and 1, assigned to a simulant *for the entirety of the simulation*
```

Comment:
I'm confused by this description:

- What is "sampled as described above based on a random number between 0 and 1" referring to?
- I thought rr_i was what gets assigned to the simulant, and rr would be a global parameter, not a property of a simulant. Do I have the wrong interpretation here?

Reply:
I clarified this in the latest commit! Let me know if it still doesn't make sense.